### PR TITLE
Show permission denied message for applications that are not public

### PR DESCRIPTION
**Pull request by ahennr**
See title.

Plz review @terrestris/devs

### CODE REVIEW

File: src/i18n/translations.ts [hunk 1]
```diff
@@ -147,7 +147,8 @@ export default {
         errorMessage: 'Fehler beim Laden der Applikation',
         errorDescription: 'Aufgrund eines unerwarteten Fehlers konnte die Applikation nicht geladen werden.',
         errorDescriptionAppIdNotSet: 'Keine Applikations-ID angegeben. Bitte geben Sie die ID als Abfrageparameter an, z.B. ?applicationId=1909',
-        errorDescriptionAppConfigNotFound: 'Die Applikation mit der ID {{applicationId}} konnte nicht geladen werden.'
+        errorDescriptionAppConfigNotFound: 'Die Applikation mit der ID {{applicationId}} konnte nicht geladen werden.',
+        permissionDeniedUnauthorized: 'Dies ist keine öffentliche Application. Anmeldung erforderlich.'
```
Comment (dnlkoch) [inline src/i18n/translations.ts:151]:
```suggestion
        permissionDeniedUnauthorized: 'Dies ist keine öffentliche Applikation. Anmeldung erforderlich.'
```

File: src/bootstrap.tsx [hunk 2]
```diff
@@ -699,12 +703,18 @@ const renderApp = async () => {
       await i18n.init(initOpts);
     }
 
+    let type: any = 'warning';
```
Comment (dnlkoch) [inline src/bootstrap.tsx:706]: Can potentially be typed via ` AlertProps['type']`.

**Review by dnlkoch — APPROVED** [review]
Nice, thanks!

File: src/bootstrap.tsx (hunk 2 above)
Comment (ahennr) [reply src/bootstrap.tsx]: Nice, fixed in 069174b

**Pull request merged**